.changeset/spotty-rice-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+During the build, pages that take more than 500 milliseconds to render, will have their time increased logged in red.
```

Comment:
```suggestion
Adds color-coding to the console output during the build to highlight slow pages. 

Pages that take more than 500 milliseconds to render will have their build time logged in red. This change can help you discover pages of your site that are not performant and may need attention.
```
Was trying to think of how to make this sound helpful and exciting! :sweat_smile: Just a thought for a direction you could take!